src/systems/filecoin_vm/sysactors/reward_actor.id

```diff
@@ -22,8 +22,10 @@ type RewardActorState struct {
 type RewardActorCode struct {
     Constructor(rt vmr.Runtime)
 
-    // put Reward in RewardMap
-    MintReward(rt vmr.Runtime)
+    // Allocates a block reward to the owner of a miner, less
+    // - penalty, which is burnt
```

Comment:
```suggestion
    // - penalty used to pay for some messages executed in blocks which is burnt
```

Or something correct. I.e. where does penalty come from?

Reply:
It comes from what would have been paid to the miner as block reward. This actor doesn't care why.